extensions/amp-date-display/0.2/date-display.js

```diff
@@ -113,17 +114,19 @@ function getEpoch(props) {
     epoch = Date.now();
   } else if (datetime) {
     epoch = Date.parse(datetime);
-    userAssert(!isNaN(epoch), 'Invalid date: %s', datetime);
+    // TODO(wassgha): Discuss a replacement for userAssert
+    // userAssert(!isNaN(epoch), 'Invalid date: %s', datetime);
```

Comment:
To note: I see these two cases as completely different, tbh: 

1. Incomplete data (e.g. absent prop). Mostly ok. If it's required some PropTypes or `d.ts` should reflect that. But from component point of view, we can just do `return null`.
2. Invalid value is specified. I actually don't see it as too harmful if it just failed for now. IMHO that could always happen: some parser or some operation would throw an error - just not predictable. So pre-throwing an error is maybe ok for this?